src/query/sql/src/planner/expression_parser.rs

```diff
@@ -141,7 +143,7 @@ pub fn parse_to_remote_string_expr(
     ctx: Arc<dyn TableContext>,
     table_meta: Arc<dyn Table>,
     sql: &str,
-) -> Result<RemoteExpr<String>> {
+) -> Result<Filters> {
```

Comment:
Why change the return type to `Filters`? It looks like the new return type doesn't fit the function name.